string image_id
uint32 radius
uint16 center_x
uint16 center_y
